domains/hazel.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "hablethedev",
+           "email": "xendyex@tutanota.com",
+           "discord": "1113284520737771621"
+        },
+    
+        "record": {
+            "CNAME": "https://hablethedev.github.io/hazelsite/"
```

you cant have https:// and / for CNAME
```suggestion
            "CNAME": "hablethedev.github.io"
```

---

oh, sorry im not very good at this one second